sdk/eventhubs/azure_messaging_eventhubs/README.md

```diff
@@ -243,16 +208,18 @@ Each message receiver can only receive messages from a single Event Hubs partiti
 
 ```rust no_run
 use async_std::stream::StreamExt;
-use azure_messaging_eventhubs::ConsumerClient;
+use azure_messaging_eventhubs::{
+    ConsumerClient, OpenReceiverOptions, StartLocation, StartPosition,
+};
 use futures::pin_mut;
 
 async fn receive_events(client: &ConsumerClient) -> Result<(), Box<dyn std::error::Error>> {
     let message_receiver = client
         .open_receiver_on_partition(
             "0",
```

Comment:
Might want to add a comment explaining why the `start_position` is set.

Something like:

```rust
// By default, an event receiver only receives new events from the event hub. To receive events from earlier, specify
// a `start_position` which represents the position from which to start receiving events. 
// In this example, events are received from the start of the partition.